datahub-web-react/src/app/settings/AccessTokens.tsx

```diff
@@ -192,6 +192,7 @@ export const AccessTokens = () => {
                     <span>{`${formattedExpireAt.toLocaleDateString()} at ${formattedExpireAt.toLocaleTimeString()} (${localeTimezone})`}</span>
                 );
             },
+            sorter: (sourceA, sourceB) => sourceA.name.localeCompare(sourceB.name),
```

Comment:
same questions here... why are you sorting by `name` for the "Expires At" column? shouldn't we be sorting based on that date for this column?